Day4/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./App.styles.css";
+import Header from "./components/Header/Header";
+import HomePage from "./pages/HomePage/HomePage";
+
+const App = () => {
+  return (
```

Comment:
App component can be returned without the return key word. 
e.g.:  const App = ()=> (<div>
...
</div>);

Reply:
Done with this change